openra/utility.py

```diff
@@ -305,12 +295,12 @@ def ReadYaml(item=False, fullpath=""):
     if fullpath == "":
         if item is False:
             return {'response': 'wrong method call', 'error': True}
-        currentDirectory = os.getcwd() + os.sep
-        path = currentDirectory + 'openra/data/maps/' + str(item.id) + '/'
+        currentDirectory = settings.BASE_DIR
```

Comment:
This variable doesn't seem to be used anywhere else, so can move into the path.join?

Reply:
Ok I'll add another commit tonight to remove the intermediate "currentDirectory" variable.

Reply:
Ok I've added a commit which does this now.